src/chains/ethereum/ethereum/src/database.ts

```diff
@@ -49,7 +49,10 @@ export default class Database extends Emittery {
   }
 
   #initialize = async () => {
-    const levelupOptions: any = { valueEncoding: "binary" };
+    const levelupOptions: any = {
+      keyEncoding: "binary",
```

Comment:
This makes it so that we don't have to stringify our keys, which are almost always, if not always, already a Buffer.